command/cli.go

```diff
@@ -147,12 +147,12 @@ func (va *FormatArgs) AddFlagSets(flags *flag.FlagSet) {
 	flags.BoolVar(&va.Check, "check", false, "check if the input is formatted")
 	flags.BoolVar(&va.Diff, "diff", false, "display the diff of formatting changes")
 	flags.BoolVar(&va.Write, "write", true, "overwrite source files instead of writing to stdout")
-
+	flags.BoolVar(&va.Recursive, "recursive", true, "Also process files in subdirectories")
```

Comment:
I think you intended to make this option false by default :
```suggestion
	flags.BoolVar(&va.Recursive, "recursive", false, "Also process files in subdirectories")
```

Reply:
Applyed change manually